documentation/api-scenario/readme.md

```diff
@@ -12,11 +12,11 @@ API Scenario is a YAML file defining RESTful API usage scenarios of your service
 
 ![demo](./how-to/runApiTest.gif)
 
-## Quick start
+## Quick starts
 
 - [Example: Write and run your first API scenario file](./how-to/QuickStart.md)
 - [Example: Generate a basic API scenario file](./how-to/generateABasicApiScenario.md)
-- [Example: Integrate with ARM Template](./how-to/integrateWithArmTemplate.md)
+- [Example: Resolve cross-RP dependencies](./how-to/resolveDependencies.md)
```

Comment:
Resolve cross-RP and external dependencies?

Reply:
fixed